### DISCUSSION TITLE
Copying schema and data failed

### DISCUSSION THREAD

**Discussion by kikohz** [planetscale/discussion#489] [Bug Reports]
Hi, I am prompted when importing AWS RDS database
1. Copy schema and data failed
Something went wrong in the initial import process. I don't know what is causing the problem, can you help me?

image is https://imgur.com/download/yqkSN5v/

thanks

**Comment by orware** [comment]
Hello @kikohz,

If you haven't done so already, please reach out via https://support.planetscale.com/ so that we may more easily work with you on this import related issue since we typically have to get into specifics and review things in more detail together.

For existing resources we will usually ask if our existing documentation pages have been reviewed and any prerequisite steps completed already:
https://planetscale.com/docs/imports/aws-rds-migration-guide
https://planetscale.com/docs/imports/database-imports
https://planetscale.com/docs/imports/import-tool-user-requirements
